spec/controllers/organizers/teams_controller_spec.rb

```diff
@@ -18,17 +18,17 @@
     include_context 'with admin logged in'
 
     describe 'GET index' do
```

Comment:
Please remove any tests and logic related to the deprecated columns and test for the new / current types of teams instead.